Singer.API/ClientApp/src/app/modules/core/components/nav-menu/nav-menu.component.ts

```diff
@@ -93,18 +115,26 @@ export class NavMenuComponent {
       // Loop through the requirements
       requirements.forEach(requirement => {
          // If a requirement is not met: result = false
-         if (!this.checkRequirement(requirement)) { result = false; }
+         if (!this.checkRequirement(requirement)) {
+            result = false; }
```

Comment:
either make it one line again or move your closing bracket to the next line plz.